keps/976-plain-pods/README.md

```diff
@@ -538,39 +539,51 @@ spec:
 Pods need to have finalizers so that we can reliably track how many of them run to completion and be
 able to determine when the Workload is Finished.
 
-A Pod-group reconciler would keep track of the groups of Pods and their respective Workload object,
-based on the `jobframework.Reconciler`.
-After a Workload is admitted, as the Pod-group reconciler ungates pods, it would keep an in-memory
-cache of expected ungated pods: the number of ungated pods that are not reflected in the informers
-yet, per Pod-group. This number decreases as the event handler observes Pods transition from being
-gated to ungated.
+The Pod reconciler will run in a "composable" mode: a mode where a Workload is composed of multiple
+objects. The `jobframework.Reconciler` will be reworked to accomodate this.
 
-In the Pod event handler, we decrement the counter when we see a transition from having
-the scheduling gate `kueue.x-k8s.io/admission` to not having it.
+After a Workload is admitted, each Pod that owns the workload enters the reconciliation loop.
+The reconciliation loop collects all the Pods that are not Failed and constructs an in-memory
+Workload. If there is an existing Workload in the cache and it has smaller Pod counters than the
+in-memory Workload, then it is considered unmatching and the Workload is evicted.
```

Comment:
What happens if a cached existing Workload has larger than pod counters than an in-memory Workload?
Will The reconciler evict the Workload the same as smaller case?

Reply:
We only evict if the counters in the Workload are smaller, not larger.

Reply:
It makes sense. Thanks.